src/test/java/networkbook/logic/commands/AddCommandTest.java

```diff
@@ -61,6 +63,23 @@ public void execute_noFieldSpecifiedUnfilteredList_success() {
         CommandTestUtil.assertCommandSuccess(addCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
+    public void execute_multipleEntriesForUniqueField_commandException() throws Exception {
+        CommandTestUtil.showPersonAtIndex(model, TypicalIndexes.INDEX_FIRST_PERSON);
+
+        Person personInFilteredList = model.getFilteredPersonList()
+                .get(TypicalIndexes.INDEX_FIRST_PERSON.getZeroBased());
+
+        AddCommand addCommand = new AddCommand(TypicalIndexes.INDEX_FIRST_PERSON,
+                new EditPersonDescriptorBuilder().withPriority(CommandTestUtil.VALID_PRIORITY_AMY).build());
+
+        assertThrows(CommandException.class, () -> {
+            addCommand.execute(model);
+            addCommand.execute(model);
```

Comment:
Would need more explanation on what this test is doing here.

Reply:
It's to verify that CommandException is thrown when adding new priority to a person already with a priority.

Let me change the JUnit assertion to check the error message instead.

Reply:
Please change the name of the method to highlight that you are testing the priority only.
Other than that, everything else looks good! Let me know when you have done.

Reply:
Yep I'm done fixing the test case, please review again, thanks!

Reply:
Oh wait, I missed one line to fail the test case

Reply:
Okay done